circle.yml

```diff
@@ -13,10 +16,21 @@ dependencies:
     - ~/.ivy2
     - ~/.sbt
     - ~/.m2
+
 test:
   override:
     - sbt scalastyle
     - sbt test
   post:
     - sbt universal:package-zip-tarball
     - cp target/universal/*.tgz $CIRCLE_ARTIFACTS
+
+deployment:
+  hub:
+    branch: /.*/
+    commands:
+      - sbt universal:package-zip-tarball
```

Comment:
You might want to check the project templates and update the circle config to be more inline with them.